pytorch_lightning/core/memory.py

```diff
@@ -213,6 +216,29 @@ def out_sizes(self) -> List:
     def param_nums(self) -> List[int]:
         return [layer.num_parameters for layer in self._layer_summary.values()]
 
+    @property
+    def total_parameters(self) -> int:
+        return sum(p.numel() for p in self._model.parameters())
+
+    @property
+    def trainable_parameters(self) -> int:
+        return sum(p.numel() for p in self._model.parameters() if p.requires_grad)
+
+    @property
+    def model_size(self) -> float:
+        return self._model_size()
+
+    def _model_size(self) -> float:
+        """
+        Calculates the total model params size in MBs.
+
+        NOTE: Currently only model params size is supported.
+
+        Returns:
+            Total params size (MB).
+        """
+        return self.total_parameters * self._precision_megabytes
```

Comment:
```suggestion
    def model_size(self) -> float:
        return self.total_parameters * self._precision_megabytes
```
any reason for separating these two??

Reply:
in future, i will be add input_size and forward pass support hence will need a separate function for better readability.

Reply:
Then let's update it then. For now I'd suggest to merge these.

Reply:
ok sure make sense, will separate it later then :)